query/filtering_lexer_test.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 func TestFilteringLexer(t *testing.T) {
-	lexer := NewFilteringLexer(`()14 13.23 'abc'"bcd" field1 and or  not == eq ne != match ~ nomatch !~ gt > ge >= lt < le <= null := ieq [1,5, 6] ['Hello','World'] in `)
+	lexer := NewFilteringLexer(`()14 13.23 'abc'"bcd" field1 and or  not == eq ne != match ~ nomatch !~ gt > ge >= lt < le <= null := ieq [1,5, 6] ['Hello','World'] in '''""' """''"`)
```

Comment:
we should document this in README as current escaping logic is not obvious.

Reply:
I would expect it's documented in REST API Syntax.